docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -152,9 +152,12 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 
 ## Markdown (`markdown`)
 
-- `ENABLE_HARD_LINE_BREAK`: **true**: Render soft line breaks as hard line breaks, which
+- `ENABLE_HARD_LINE_BREAK_IN_COMMENTS`: **true**: Render soft line breaks as hard line breaks in comments, which
```

Comment:
```suggestion
Note: before 1.13, a single setting (`ENABLE_HARD_LINE_BREAK`) was used to control the following two cases simultaneously. It was deprecated in favor of better granularity.
- `ENABLE_HARD_LINE_BREAK_IN_COMMENTS`: **true**: Render soft line breaks as hard line breaks in comments, which
```

Reply:
It's not even deprecated - it's just not used.

Reply:
As this was introduced in 1.12 we still have time to fix this to not be breaking

Reply:
oh cool, let's remove that comment then!

Reply:
Default was changed at least, setting was there before but was false by default imho

Reply:
Yes it was https://github.com/go-gitea/gitea/pull/11162